### DISCUSSION TITLE
Release 24.4 of Mojo 🔥 and MAX ⚡ now with quantization and macOS support is live!

### DISCUSSION THREAD

**Discussion by jackos** [modular/modular#2981] [Announcements]
Release 24.4 of Mojo 🔥 and MAX ⚡ now with quantization and macOS support is live!

Check out the new [MAX install guide which includes Mojo](https://modul.ar/install-max) or the [Mojo standalone install guide](https://docs.modular.com/mojo/manual/get-started/).

Read the [launch blog](https://modul.ar/24-4) and an example running [llama3 with MAX and quantization](https://modul.ar/llama3).

Find the epic changelogs here: [Mojo](https://modul.ar/mojo-changelog) and [MAX](https://modul.ar/max-changelog).

Huge thanks to the community for the 200+ contributions to the standard library this release!

If you've had troubles installing with Python self-tests in the past, Mojo now links to first Python version on top of PATH when `MOJO_PYTHON_LIBRARY` is **not** set. You can activate a virtual environment of a compatible Python installation and try again, see the script for [finding a compatible environment on your system](https://modul.ar/fix-python).

**Comment by hchaudhary1** [comment]
does mojo/max use MLX on Apple M silicon?

**Comment by fanfantingfeng** [comment]
I cannot install it by the command "modular install mojo", it is always overtime. The modular package website is not friendly.
Is that possible to apply another way to install mojo without VPN?
